ipld/amt/src/node.rs

```diff
@@ -553,6 +553,7 @@ where
         Ok((true, did_mutate))
     }
 
+    #[allow(clippy::too_many_arguments)]
```

Comment:
if this is ok for an internally used function I'll leave it as is, otherwise could refactor `start_at and limit` into one tuple or compress other parameters